File: src/main/java/no/nav/veilarbvedtaksstotte/config/KafkaEnvironmentContext.java
package no.nav.veilarbvedtaksstotte.config;

import lombok.Data;
import lombok.experimental.Accessors;

import java.util.Properties;

@Data
@Accessors(chain = true)
public class KafkaEnvironmentContext {
    Properties onPremConsumerClientProperties;
    Properties onPremProducerClientProperties;
    Properties aivenConsumerClientProperties;
    Properties aivenProducerClientProperties;
}

File: src/main/java/no/nav/veilarbvedtaksstotte/config/KafkaEnvironmentContext.kt
package no.nav.veilarbvedtaksstotte.config

import java.util.Properties

class KafkaEnvironmentContext(
    val onPremConsumerClientProperties: Properties,
    val onPremProducerClientProperties: Properties,
    val aivenConsumerClientProperties: Properties,
    val aivenProducerClientProperties: Properties
)
